src/pages/Homepage/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import {Form, Repo} from '../../components';
+import{ Header } from '../../layout';
+
+
+const Homepage = () => {
+
+    const error = useSelector(state => state.error);
+    const avatarLink = useSelector(state => state.avatar);
+   
+    return (
+        <>
+        <Header />
+        <Form />
+        { avatarLink ? <img src={avatarLink} alt="avatar image" style={{width:"5rem"}} /> : <span id="no-image-here"></span> } 
+        { error ? <p role='alert'>Oops!  We cannot find that user.  Try a different name.</p> : <Repo /> }
+        </>
+    )
+}
+
+export default Homepage;
```

Comment:
Glad you added an error message and it works as expected. Although if you view a existing user then try search for a user that does not exist, it keeps the old avatarLink. Maybe you could wrap the avatarLink in the error conditional operator somehow? I don't know if that would cause issues

Reply:
Well spotted - I'd not noticed this until testing out the other bug you found!  We considered moving the avatar rendering to a completely separate component anyway, but didn't have time to implement, so maybe we should push it to the top of our to do list!